debian/source/lintian-overrides

```diff
@@ -2,5 +2,8 @@
 ubuntu-advantage-tools: missing-build-dependency-for-dh_-command dh_systemd_start => dh-systemd
 ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd => dh-systemd
 
+# Lintian doesn't see the second debhelper dependency with >13.3 on jammy++
+ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd *debhelper* (>= 9.20160709~)*
```

Comment:
This is correct and overrides the lintian error. There is an alternative that would avoid the almost duplicate with line 3:

```
ubuntu-advantage-tools: missing-build-dependency-for-dh-addon  *systemd* 
```

With that, line three could be removed and this new override would cover both cases. I understand you want to minimize the changes here to release version 28, but I thought I should at least raise this here :)

Feel free to land this as you proposed and fix it later as part of #2608 .

Reply:
This is closely related enough I think we can include it now! I'll test this locally and update the PR with it

Reply:
@lucaskanashiro updated!

Reply:
Perfect, thanks @orndorffgrant !